### PR TITLE
Add Ask.com.xml

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

**Comment by J0WI** [comment]
@gloomy-ghost could you help me to get this merged?

File: src/chrome/content/rules/Ask.com.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<!--
+	Nonfunctional hosts in *.ask.com:
+		- about.ask.com (r)
+		- at.ask.com (m)
+		- br.ask.com (m)
+		- ca.ask.com (m)
+		- ext.ask.com (m)
+		- feedback.ask.com (m)
+		- id.ask.com (m)
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/Ask.com.xml]: Not in DNS

File: src/chrome/content/rules/Ask.com.xml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+		- mx.ask.com (m)
+		- pt.ask.com (m)
+		- ru.ask.com (m)
+		- safesearch.ask.com (m)
+		- search.ask.com (m)
+			- nortonsafe.search.ask.com (mixed content)
+			- secure.search.ask.com (m)
+			- www.search.ask.com (mixed content)
+		- social.ask.com (expired)
+		- sp.ask.com (m)
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/Ask.com.xml]: Not in DNS

File: src/chrome/content/rules/Ask.com.xml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+		- tb.ask.com (m)
+			- home.tb.ask.com (m)
+			- akd.search.tb.ask.com (m)
+		- www.ask.com (m)
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/Ask.com.xml]: WFM

File: src/chrome/content/rules/Ask.com.xml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+-->
+<ruleset name="Ask.com">
+	<target host="ask.com" />
+	<target host="de.ask.com" />
+	<target host="dp.tb.ask.com" />
+	<target host="es.ask.com" />
+	<target host="fr.ask.com" />
+	<target host="help.ask.com" />
+	<target host="hp.ask.com" />
+	<target host="int.search.tb.ask.com" />
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/Ask.com.xml]: sorting order?

**Comment by J0WI** [comment]
Thanks, I've updated the PR.

**Pull request merged**